### PR TITLE
Fix run mockup

### PR DESCRIPTION
**Pull request by jordiandreu**
Fix mockup applicaion, paired with https://github.com/mxcube/HardwareRepository/pull/497

### CODE REVIEW

File: gui/bricks/InstanceListBrick.py [hunk 1]
```diff
@@ -447,10 +447,13 @@ def instance_initializing(self):
         BaseWidget.set_instance_location(local)
 
         for widget in QtImport.QApplication.allWidgets():
-            if hasattr(widget, "configuration"):
-                active_window = widget
-                break
-
+            try:
```
Comment (rhfogh) [inline gui/bricks/InstanceListBrick.py:450]: Again, this should not return an error. Do you really get name_error form a hasattr call??

File: gui/bricks/InstanceListBrick.py [hunk 2]
```diff
@@ -447,10 +447,13 @@ def instance_initializing(self):
...
+                if hasattr(widget, "configuration"):
```
Comment (rhfogh) [inline gui/bricks/InstanceListBrick.py]: See above

File: gui/utils/GUIDisplay.py [hunk 3]
```diff
@@ -230,15 +230,19 @@ def set_exp_mode(self, state):
...
-                if hasattr(widget, "set_expert_mode"):
-                    # if isinstance(widget, BaseWidget):
-                    widget.setWhatsThis("")
-                    try:
-                        widget.set_expert_mode(False)
-                    except BaseException:
-                        logging.getLogger().exception(
-                            "Could not set %s to user mode" % widget.objectName()
-                        )
+                try:
```
Comment (rhfogh) [inline gui/utils/GUIDisplay.py]: See above

File: gui/bricks/InstanceListBrick.py (hunk 1 above)
Comment (jordiandreu) [reply gui/bricks/InstanceListBrick.py:450]: The error comes from _pyqtgraph_. The __getattr__ method from PlotWidget raises the exception:

>     def __getattr__(self, attr):  ## implicitly wrap methods from plotItem
>        if hasattr(self.plotItem, attr):
>            m = getattr(self.plotItem, attr)
>            if hasattr(m, '__call__'):
>               return m
>        raise NameError(attr)
Any other workaroud/solution is welcome.

Comment (rhfogh) [reply gui/bricks/InstanceListBrick.py:450]: That is just totally weird!! Looks like it is a bug in PlotWidget, which should raise AttributeError, not NameError. I cannot think of any other solution either.

**Review by rhfogh — APPROVED** [review]

File: gui/utils/GUIDisplay.py [hunk 4]
```diff
@@ -230,15 +230,19 @@ def set_exp_mode(self, state):
...
+                    if hasattr(widget, "set_expert_mode"):
```
Comment (IvarsKarpics) [inline gui/utils/GUIDisplay.py:234]: is this try again for the pyqtgraph widget?

Comment (jordiandreu) [reply gui/utils/GUIDisplay.py:234]: Yes, indeed.

**Pull request merged**